src/components/Constants/UseApiConstants.tsx

```diff
@@ -1,7 +1,12 @@
-import { useLoadConstantsQuery } from './LoadConstants.generated';
+import {
+  LoadConstantsQuery,
+  useLoadConstantsQuery,
+} from './LoadConstants.generated';
 
 // Use this call to force loading from cache storage first
-export const useApiConstants = () => {
+export const useApiConstants = ():
+  | LoadConstantsQuery['constant']
+  | undefined => {
```

Comment:
TypeScript couldn't infer these types?

Reply:
No, it was throwing a warning not an error about not having a return type. And for some reason, it couldn't infer that it could return ```undefined``` 🤷‍♂️